src/view/screens/Search/Search.tsx

```diff
@@ -684,6 +684,12 @@ export function SearchScreen(
                 })
               } else {
                 setShowAutocomplete(true)
+                if (isIOS) {
+                  // We rely on selectTextOnFocus, but it's broken on iOS:
+                  // https://github.com/facebook/react-native/issues/41988
+                  textInput.current?.setSelection(0, searchText.length)
```

Comment:
It's not ideal that we rely on `searchText.length` here. I think in theory it could be stale. I haven't been able to trigger that tho. Seems fine.